src/App.js

```diff
@@ -67,6 +67,10 @@ function App() {
 
   var permission = "";
 
+  if(localStorage.getItem("permission")) {
+    permission = localStorage.getItem("permission");
+  }
+
   if (data && data.getUser)
```

Comment:
is the if block on line 74 necessary anymore? If it isn't doing anything could you remove it? Otherwise everything is good here. Thanks, Jesus